### ISSUE TITLE
DahuaVTO2MQTT Docker  - install error

### ISSUE THREAD

**Issue by stonehav** [hassio-addons/addon-portainer#58]
# Problem/Motivation

> I am trying to install the DahuaVTO2MQTT  Docker  within the HA add on Portainer 
(https://community.home-assistant.io/t/dahua-vto-to-mqtt-broker/169781)

I followed the instuctions in this video- https://www.youtube.com/watch?v=-oOP38KpvfE

The container creates successfully , however i cannot see the VTO events within HA
I can see these error in the container logs. 

standard_init_linux.go:219: exec user process caused: exec format error,
standard_init_linux.go:219: exec user process caused: exec format error,
standard_init_linux.go:219: exec user process caused: exec format error,
standard_init_linux.go:219: exec user process caused: exec format error,
standard_init_linux.go:219: exec user process caused: exec format error,
standard_init_linux.go:219: exec user process caused: exec format error,
standard_init_linux.go:219: exec user process caused: exec format error,
standard_init_linux.go:219: exec user process caused: exec format error,
standard_init_linux.go:219: exec user process caused: exec format error,
standard_init_linux.go:219: exec user process caused: exec format error,
standard_init_linux.go:219: exec user process caused: exec format error,
standard_init_linux.go:219: exec user process caused: exec format error,

## Expected behavior

> I am expecting to see the events published within HA

## Actual behavior

> I do not see anything within HA events or in the Mqtt broker. 

## Steps to reproduce

1. Running HA OS 5.10
2. Mosquitto broker 5.1
3. Portainer 1.24.20 add on 
    Protection Mode off 
    no configuration changes
3.  create container using below  Variables
version: '2'
services:
  dahuavto2mqtt:
    image: "eladbar/dahuavto2mqtt:latest"
    container_name: "dahuavto2mqtt"
    hostname: "dahuavto2mqtt"
    restart: always
    environment:
      - DAHUA_VTO_HOST=vto-host
      - DAHUA_VTO_USERNAME=Username
      - DAHUA_VTO_PASSWORD=Password
      - MQTT_BROKER_HOST=mqtt-host
      - MQTT_BROKER_PORT=1883
      - MQTT_BROKER_USERNAME=Username
      - MQTT_BROKER_PASSWORD=Password 
      - MQTT_BROKER_TOPIC_PREFIX=DahuaVTO
      - DEBUG=False
 
## Proposed changes

Hoping somebody can help/provide direction. 

Thanks

**Comment by stonehav** [comment]
Thanks for the quick response. 

Do you know of any workarounds ?

**Comment by frenck** [comment]
It is not about workarounds. The container you are trying to download is not made for the CPU you are running on.

You are basically asking if I know a workaround to run your steam powered vehicle on electricity...

**Comment by stonehav** [comment]
Thank you for replying.

Get Outlook for iOS <https://aka.ms/o0ukef>
------------------------------
*From:* Franck Nijhof <notifications@github.com>
*Sent:* Thursday, January 21, 2021 4:19:51 PM
*To:* hassio-addons/addon-portainer <addon-portainer@noreply.github.com>
*Cc:* stonehav <alexrobertfield@gmail.com>; Author <
author@noreply.github.com>
*Subject:* Re: [hassio-addons/addon-portainer] DahuaVTO2MQTT Docker -
install error (#58)


It is not about workarounds. The container you are trying to download is
not made for the CPU you are running on.

You are basically asking if I know a workaround to run your steam powered
vehicle on electricity...

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/hassio-addons/addon-portainer/issues/58#issuecomment-764763114>,
or unsubscribe

**Comment by stonehav** [comment]
Thanks for replying.

**Comment by stonehav** [comment]
I have been looking at alternative methods to get HA to receive messages from my  Dahua VTO. 
everthing returns me back to this - https://community.home-assistant.io/t/dahua-vto-to-mqtt-broker/169781

is the reason i can't access the container because my hardware is Home Assistant Blue?
would the add on mentioned (that is no longer maintained) work? https://github.com/intermittech/hassio-addons

Could you please help me understand? in laymans terms please.

**Comment by dvbit** [comment]
@stonehav you are very likely installing the container on a raspberry PI. I think it is suited for x86 architecture
